packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -247,6 +248,35 @@ export async function testAppConfigExtensions(emptyConfig = false): Promise<Exte
   return extension
 }
 
+export async function testAppAccessConfigExtension(emptyConfig = false): Promise<ExtensionInstance> {
+  const configuration = emptyConfig
+    ? ({} as unknown as BaseConfigType)
+    : ({
+        access: {
+          admin: {direct_api_mode: 'online'},
+        },
+        access_scopes: {
+          scopes: 'read_products,write_products',
+          use_legacy_install_flow: true,
+        },
+        auth: {
+          redirect_urls: ['https://example.com/auth/callback'],
+        },
+      } as unknown as BaseConfigType)
+
+  const allSpecs = await loadLocalExtensionsSpecifications()
```

Comment:
I just duplicated the function from above 😅 
There is no need to load all extensions. We originally had this because you couldn't import a single specification (they weren't "exported") so you had to load all.
But that has changed and is now possible to import specific ones